src/desktop/components/form/messenger.coffee

```diff
@@ -43,6 +43,9 @@ module.exports = class Messenger
       else
         $el.append "<span class='is-error'>#{message}</span>"
 
+        # remove after gdpr compliance test closes
+        $('.gdpr-signup__form__checkbox__accept-terms .is-error').hide() 
```

Comment:
#minor, but its usually better to use `@$('..')` (or @$el.find(...)`), so that the jQuery selector is scoped to the `@$el`.

In this case, you're looping thru `inline`, but hiding a globally-scoped element (using `$`). I'm a bit unfamiliar, but perhaps you meant to move this line _outside_ of the `map`? In which case it would probably be `@$form('...')`.

Reply:
Yeah good call. Yeah it should probably live outside of the map, but in the render function.